### PR TITLE
Cacie/chore/upgrade electron 27

### PR DESCRIPTION
**Pull request by cacieprins**
- Closes 

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/electron/README.md [hunk 1]
```diff
@@ -45,19 +45,44 @@ Upgrading `electron` involves more than just bumping this package's `package.jso
     - The Ubuntu images in [base-internal](https://github.com/cypress-io/cypress-docker-images/tree/master/base-internal) are updated to be used in the [system binary tests](../../system-tests/test-binary) if any of the following are true for the images used inside the system binary tests:
       - The last two major [Ubuntu LTS Releases](https://ubuntu.com/about/release-cycle) are out-of-date.
       - The [NodeJS](https://nodejs.org/en) version is not the active LTS.
+- [ ] **Update `workflows.yml`**
+    - [ ] Ensure it references the new `base-internal` and `browsers-internal` Docker images
+    - [ ] Ensure the new Electron version is used as a build target in the `Build better-sqlite3 for CentOS 7` step
+    - [ ]
```
Comment (MikeMcC399) [inline packages/electron/README.md:51]:
```suggestion
```

It looks like this line should be removed.

File: packages/electron/README.md [hunk 2]
```diff
@@ -45,19 +45,44 @@ Upgrading `electron` involves more than just bumping this package's `package.jso
...
+
+
 - [ ] **Manually smoke test `cypress open`.** Upgrading Electron can break the `desktop-gui` in unexpected ways. Since testing in this area is weak, double-check that things like launching `cypress open`, signing into Cypress Cloud, and launching Electron tests still work.
+- [] **Manually smoke test `cypress run` in record mode** Upgrading Electron can cause `better-sqlite3` to SIGSEGV the Electron process.
```
Comment (MikeMcC399) [inline packages/electron/README.md]:
```suggestion
- [ ] **Manually smoke test `cypress run` in record mode** Upgrading Electron can cause `better-sqlite3` to SIGSEGV the Electron process.
```

File: cli/CHANGELOG.md [hunk 3]
```diff
@@ -20,11 +20,15 @@ _Released 1/16/2024 (PENDING)_
 
 - Remove dependency on `@types/node` package. Addresses [#28473](https://github.com/cypress-io/cypress/issues/28473).
 - Updated  `@cypress/unique-selector` to include a performance optimization. It's possible this could improve performance of the selector playground. Addressed in [#28571](https://github.com/cypress-io/cypress/pull/28571).
+- Upgraded `electron` from `25.8.4` to `27.1.3`
+- Upgraded bundled Node.js version from `18.15.0` to `18.17.0`
+- Upraded bundled Chromium version from `114.0.5735.289` to `116.0.5845.228`
```
Comment (MikeMcC399) [inline cli/CHANGELOG.md]:
```suggestion
- Upgraded bundled Chromium version from `114.0.5735.289` to `116.0.5845.228`
```

File: packages/electron/README.md (hunk 1 above)
Comment (cacieprins) [reply packages/electron/README.md:51]: Thanks, Mike! Still making sure all of the rest of things are working as expected, so this file will likely change some more before it's marked ready for review.

Comment (MikeMcC399) [reply packages/electron/README.md:51]: Sure. I'll hold back until the PR is marked for review.

File: cli/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,15 @@
...
 ## 13.6.4
 
+_Released 1/30/2024 (PENDING)_
+
+** Dependency Updates:**
+
+- Updated  `@cypress/unique-selector` to include a performance optimization. It's possible this could improve performance of the selector playground. Addressed in [#28571](https://github.com/cypress-io/cypress/pull/28571).
+- Upgraded `electron` from `25.8.4` to `27.1.3`
+- Upgraded bundled Node.js version from `18.15.0` to `18.17.0`
+- Upgraded bundled Chromium version from `114.0.5735.289` to `116.0.5845.228`
```
Comment (emilyrohrbough) [inline cli/CHANGELOG.md]: Chromium version is `v18.17` --  https://www.electronjs.org/docs/latest/tutorial/electron-timelines / showing in System Tests output as well

File: package.json [hunk 5]
```diff
@@ -211,7 +212,7 @@
     "yarn-deduplicate": "3.1.0"
   },
   "engines": {
-    "node": ">=18.15.0",
+    "node": ">=18.17.0",
```
Comment (emilyrohrbough) [inline package.json:215]: This would be non-passive to bump for yarn projects

Comment (emilyrohrbough) [reply package.json:215]: wait -- my bad -- this is our for our development version. This is fine. I mistook this for cli/package.json.

**Comment by cacieprins** [comment]
Closing this PR due to botched merge/rebase- PR that continues the process at https://github.com/cypress-io/cypress/pull/28792

**Pull request closed without merging**